tensorflow/core/grappler/optimizers/constant_folding_test.cc

```diff
@@ -551,9 +551,9 @@ TEST_F(ConstantFoldingTest, ConstantPushDownBiasAdd) {
 TEST_F(ConstantFoldingTest, MulConvPushDownTest_Conv2D_ScalarConst) {
   for (string data_format : {
          "NHWC",
-#if GOOGLE_CUDA
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Comment:
@rmlarsen is there some joint declaration this could be guarded on? (E.g., TF_USE_GPU?) Or an attribute? (E.g., SUPPORTS_NCHW ? - I'm assuming there is not something _that_ specific :) )

Reply:
We can also address this in follow up, it might likely be there is not something that exactly first this (esp as Christian was happy with the change).